### PR TITLE
Setup JS toolchain for Tailwind and DaisyUI

### PR DESCRIPTION
**Pull request by anirudhprabhakaran3**
# Description

Setup JS toolchain for Tailwind and DaisyUI.

Fixes # (issue)

## Dependencies

List any dependencies that are required for this change.

## Type of Change

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration. (for bug fix / breaking change)
_Put an `x` in the boxes that apply_

- [ ] Test A
- [ ] Test B

---

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by nishant-nayak** [comment]
Is this something we want to add to dev environments? I definitely see the benefit in adding this toolchain for production releases, but unless there's a way to "hot reload" the Tailwind CSS definitions each time a change is made to frontend code I don't think we should add this to the development toolchain.

**Comment by anirudhprabhakaran3** [comment]
Aah yes, there is. In the README instrucetion given, instead of using `--minify` we just have to use `--watch` and it will automatically reload everything.

File: README.md [hunk 1]
```diff
@@ -4,19 +4,31 @@ A Django based web application to manage all things IEEE NITK.
...
-1. Run the following command to create a copy of the environment file. Change the environment variables as necessary in the copied file.
+1. Run the following command to create a copy of the environment file. Change the environment variables as necessary in
+   the copied file.
 
 ```sh
 cp env.example .env
 ```
 
-2. Run docker compose
+2. You'll need to setup the JS Toolchain, so that Tailwind and DaisyUI work properly. Install the version of `node`
```
Comment (nishant-nayak) [inline README.md:14]: This is no longer relevant right? Update the docs accordingly. Mention that for Windows, this hot reloading only works with WSL. For non-WSL, I think they can either use these steps itself, or restart the container on each change.

**Review by nishant-nayak — CHANGES_REQUESTED** [review]
Update the README, I think the change itself works as expected. Nice addition!

**Review by nishant-nayak — APPROVED** [review]
LGTM!

**Pull request merged**